File: 1137-n-th-tribonacci-number/1137-n-th-tribonacci-number.cs
public class Solution {
    public int Tribonacci(int n) {
        if (n == 0) return 0;
        int first = 0;
        int second = 1;
        int third = 1;
        for (int i = 3; i <=n; i++){
            int next = first + second + third;
            first = second;
            second = third;
            third = next;
        }
        return third;
    }
}

File: 1137-n-th-tribonacci-number/1137-n-th-tribonacci-number.js
var tribonacci = function (n) {
    if (n === 0) return 0;
    let first = 0;
    let second = 1;
    let third = 1;
    for (i = 3; i <= n; i++) {
        let next = first + second + third;
        first = second;
        second = third;
        third = next;
    }
    return third;
};